README.md

```diff
@@ -168,6 +168,20 @@ kdebug -t vmrebootdetector \
     --vmrebootdetector.checkdays=100
 ```
 
+### Package upgrade inspect
+
+Check upgraded packages within last 14 days:
+
+```
+kdebug --tool upgradeinspector --upgradeinspector.checkdays 14
```

Comment:
I just made some changes to CLI options yesterday. Now you can simply use `kdebug -t upgradeinspector --checkdays 14`

Reply:
Aligned with latest changes